kura/org.eclipse.kura.core.net/src/main/java/org/eclipse/kura/core/net/util/NetworkUtil.java

```diff
@@ -72,11 +50,7 @@ public static short getNetmaskShortForm(String netmask) throws KuraException {
             throw new KuraException(KuraErrorCode.INTERNAL_ERROR, "netmask is null");
```

Comment:
Needs to be `KuraRuntimeException`. There already exists an utility class `Preconditions` which performs all these necessary checks. You can reuse that.